devel.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update && apt-get install -y qt5-default libqt5webkit5-dev \
 
 COPY vendor/ ./vendor
 COPY Gemfile* ./
-COPY entrypoint.sh ./
+COPY devel.entrypoint.sh ./entrypoint.sh
```

Comment:
Ok, I think I figured out what the issue is! Please just copy this directly to `./` because we volume mount the compliance-backend repo into this devel container, and then it runs the prod entrypoint.sh. You'll also need to update line 15, which specifies the ENTRYPOINT.

Reply:
I guess this is not needed as it serves the purpose of properly renaming the entrypoint.

Reply:
Again, copying this to `./entrypoint.sh` and setting the Dockerfile to use `./entrypoint.sh` will not work in devel because we volume mount the entire repo.